assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -78,54 +78,54 @@ che.workspace.probe_pool_size=10
 
 # RAM limit default for each machine that has no RAM settings in its environment.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# Value less than or equal to `0` is interpreted as disabling the limit.
 che.workspace.default_memory_limit_mb=1024
 
 # RAM request for each container that has no explicit RAM settings in its environment.
 # This amount is allocated when the workspace container is created.
 # This property may not be supported by all infrastructure implementations.
 # Currently it is supported by {orch-name}.
 # A memory request exceeding the memory limit is ignored, and only the limit size is used.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# Value less than or equal to `0` is interpreted as disabling the limit.
 che.workspace.default_memory_request_mb=200
 
 # CPU limit for each container that has no CPU settings in its environment.
-# Specify either in floating point cores number, for example, `0.125`,
-# or using the Kubernetes format, integer millicores, for example, `125m`.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# Specify the limit either in floating point cores number, for example: `0.125`,
+# or by using the Kubernetes format, integer millicores, for example: `125m`.
+# Value less than or equal to `0` is interpreted as disabling the limit.
 che.workspace.default_cpu_limit_cores=-1
 
-# CPU request for each container that has no CPU settings in environment.
-# A CPU request exceeding the CPU limit is ignored, and only limit number is used.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# CPU request for each container that has no CPU settings in its environment.
+# A CPU request exceeding the CPU limit is ignored and only the limit number is used.
+# Value less than or equal to `0` is interpreted as disabling the limit.
 che.workspace.default_cpu_request_cores=-1
 
 # RAM limit for each sidecar that has no RAM settings in the {prod-short} plug-in configuration.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# Value less than or equal to `0` is interpreted as disabling the limit.
 che.workspace.sidecar.default_memory_limit_mb=128
 
 # RAM request for each sidecar that has no RAM settings in the {prod-short} plug-in configuration.
 che.workspace.sidecar.default_memory_request_mb=64
 
 # CPU limit default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration.
-# Specify either in floating point cores number, for example, `0.125`,
-# or using the Kubernetes format, integer millicores, for example, `125m`.
-# Value less or equal to 0 is interpreted as disabling the limit.
+# Specify the limit either in floating point cores number, for example: `0.125`,
+# or by using the Kubernetes format, integer millicores, for example: `125m`.
```

Comment:
see comment above. This is an awkward sentence with two examples not combined together.